javascript/javascript1/week1/homework1JS.js

```diff
@@ -0,0 +1,50 @@
+/*Age-ify 
+(A future age calculator)*/
+
+var yearOfBirth = 1995;
+var yearFuture = 2033;
+var age = yearFuture - yearOfBirth ;
+
+console.log ("You will be " + age + " in" + yearFuture + ".");
+
+
+/*Goodboy-Oldboy 
+(A dog age calculator)*/
+
+
+var dogYearOfBirth = 2018;
+var dogYearFuture = 2023;
+var humanAge = dogYearFuture - dogYearOfBirth;
+var dogYear = humanage * 7;
```

Comment:
```suggestion
var dogYear = humanAge * 7;
```

Be careful how you reference variables by name. Make sure you use the same